website/docs/advanced/gsn/setup-paymaster.md

```diff
@@ -0,0 +1,82 @@
+---
+id: setup-paymaster
+title: Creating a Paymaster
+sidebar_label: Creating a Paymaster
+---
+
+A [Paymaster](https://docs.opengsn.org/learn/index.html#paymaster) holds ETH and can implement any business logic to decide whether to accept or reject a meta transaction. For example, accepting only transactions by whitelisted users, or to the contracts methods required for onboarding users, or only transactions that include a repayment in tokens to the Paymaster, etc.
+
+A Paymaster is useful for any party willing to pay for a relayed transaction to a given address using the GSN network.
+
+## Implementation of Paymaster
+
+We provided a implementation of the Paymaster which allows for multiple payable address set by you.
+
+You can refer to the source code of our Paymaster [here](https://github.com/Open-Attestation/document-store/blob/master/contracts/NaivePaymaster.sol).
+
+## Deploying Paymaster
+
+### Pre-requisite
+
+- [OpenAttestation CLI](/docs/component/open-attestation-cli) installed
+- Private key to an Ethereum wallet with sufficient ethers or a [wallet](/docs/verifiable-document/wallet)
+
+### Deploying via OpenAttestation CLI
+
+Simply run the following command. You may replace the `<paymaster-name>` with a suitable name.
+
+```bash
+open-attestation deploy paymaster  "My Paymaster" --network ropsten --encrypted-wallet-path wallet.json
+```
+
+This will deploy the paymaster on the `ropsten` network. You should see a similar output when the deployment is successful:
+
+```bash
+ℹ  info      Deploying paymaster My Paymaster
+? Wallet password [hidden]
+…  awaiting  Decrypting Wallet [====================] [100/100%]
+ℹ  info      Wallet successfully decrypted
+...
+…  awaiting  Waiting for transaction 0xf4a222c9bcc31ebd202a110568a7798218477482b773f49290e1df8b4936a313 to be mined
+✔  success   Paymaster My Name deployed at 0xC234Fb1F1ef0ABCD1faC90ad12F4DfC97D583F95
+```
+
+> Save YOUR paymaster address for future reference
+
+After deploying your paymaster, you will need to fund it so that the paymaster would be able to pay for relayed transactions made. We can use this command to transfer 1 ETH to the paymaster for now. Replace `<paymaster-address> `with your own paymaster's address and `<value>` with the amount you want to send.
+
+```bash
+npx @ethersproject/cli --account wallet.json --network ropsten send 0xC234Fb1F1ef0ABCD1faC90ad12F4DfC97D583F95 1
```

Comment:
could you create an issue so that we see what they use and create a fund function ? for instance when you create a weallet there is an option to add funds automatically and curretly it uses a faucet, but last time the faucet went down

Reply:
https://github.com/Open-Attestation/open-attestation-cli/issues/107